jsons/1.12.1-rel.json

```diff
@@ -38,9 +38,28 @@
       "clientreq":true
     },
     {
-      "name": "jline:jline:2.13",
+      "name": "net.minecraftforge:log4j-server-fix:2.8.1",
       "url" : "http://files.minecraftforge.net/maven/",
-      "checksums" : [ "2d9530d0a25daffaffda7c35037b046b627bb171" ],
+      "checksums" : [ "TODO" ],
+      "clientreq":false,
+      "serverreq":true
+    },
+    {
+      "name": "org.jline:jline:3.4.0",
```

Comment:
Needs to be uploaded to Forge Maven: https://repo1.maven.org/maven2/org/jline/jline/3.4.0/jline-3.4.0.jar